src/components/Dropdown/Dropdown.tsx

```diff
@@ -141,12 +133,14 @@ export const Dropdown: FC<DropdownProps> = ({
 
     const showClear = !!activeItem && !!onClear;
 
-    const getDropdownBottomPosition = (dropDownSize: string) => {
-        return dropDownSize === DropdownSize.Small ? "tw-mb-2 tw-bottom-[34px]" : "tw-mb-2 tw-bottom-[60px]";
-    };
+    const leftPosition =
+        triggerRef.current && overlayRef.current
+            ? triggerRef.current?.getBoundingClientRect().right - overlayRef.current?.getBoundingClientRect().width
+            : "auto";
+    const bottomPosition = triggerRef.current ? triggerRef.current?.getBoundingClientRect().top - 8 : "auto";
 
     return (
-        <div className="tw-relative tw-w-full tw-font-sans tw-text-s">
+        <div className="tw-w-full tw-font-sans tw-text-s">
```

Comment:
The validation loader circle is off since we changed positioning here.
![Screenshot 2022-04-11 at 10 28 15](https://user-images.githubusercontent.com/3276065/162698916-9975307e-8956-457e-b0b5-cdd6f4dd9233.png)